### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 286010056**
the full path is "cd fabric-samples/commercial-paper/organization/magentocorp
./magnetocorp.sh"      miss organization

### CODE REVIEW

**Comment by btl5037** [comment]
Please follow instructions here to sign your commit

**Comment by btl5037** [comment]
Once change #138 is merged you'll need to do a rebase on the master branch to fix this issue.

**Comment by btl5037** [comment]
#138 was merged, please rebase on the tip of master

File: commercial-paper/README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ The contract code is available as either JavaScript, Java or Go. You can use eit
 In your 'MagnetoCorp' window run the following commands, to show the shell environment variables needed to act as that organization.
 
 ```
-cd fabric-samples/commercial-paper/magentocorp
+cd fabric-samples/commercial-paper/organization/magentocorp
```
[the remark this answers is not in the document]
Comment (ryjones) [reply commercial-paper/README.md]: done

**Comment by ryjones** [comment]
@lehors change made

**Pull request merged**